install/uninstall.go

```diff
@@ -48,6 +49,16 @@ func (k *K8sUninstaller) Log(format string, a ...interface{}) {
 func (k *K8sUninstaller) Uninstall(ctx context.Context) error {
 	k.autodetect(ctx)
 
+	for _, kind := range []string{
+		check.KindEchoName,
+		check.KindClientName,
+		check.KindPerfName,
+		check.KindHostNetNS,
+	} {
+		k.Log("🔥 Deleting %s pods in %s namespace...", kind, k.params.TestNamespace)
```

Comment:
Why not omit the `LabelSelector` altogether? That will cause the deletion of all pods under the namespace, without the need to explicitly list them (I fear that it's easy for this list to go out of sync in case we need to add some new pod in the future)